daemon/src/main/java/org/jboss/fuse/mvnd/daemon/Server.java

```diff
@@ -93,7 +93,7 @@ public Server(String uid) throws IOException {
             cli = new DaemonMavenCli();
 
             registry = new DaemonRegistry(layout.registry());
-            socket = ServerSocketChannel.open().bind(new InetSocketAddress(0));
+            socket = ServerSocketChannel.open().bind(new InetSocketAddress(Inet4Address.getLoopbackAddress(), 0));
```

Comment:
Same remark as in `DaemonConnector`: https://github.com/mvndaemon/mvnd/pull/124/files#r510083485